src/main/java/Task.java

```diff
@@ -0,0 +1,28 @@
+public class Task {
+    private final String description;
+    private boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public void markDone(){
+        this.isDone = true;
+    }
+
+    public void unmarkDone(){
+        this.isDone = false;
+    }
```

Comment:
Is it necessary to have two separate functions? Could it be done in one function to reduce class length?